app/js/classAdd.controller.js

```diff
@@ -1,7 +1,7 @@
 angular
     .module('conceptDictionaryApp')
     .controller('ClassAddController', ['$location', 'openmrsRest',
-    function($location, openmrsRest){
+    function($location, openmrsRest, $translate, $translatePartialLoader) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$translatePartialLoader" is defined but never used](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2319554390/issues/source?bid=3196969&fileBranchId=3199483#l4)